libs/mpf/src/mpf_rtp_stream.c

```diff
@@ -377,6 +377,29 @@ static apt_bool_t mpf_rtp_stream_media_negotiate(mpf_rtp_stream_t *rtp_stream)
 			codec_list2 = &local_media->codec_list;
 			codec_list1 = &remote_media->codec_list;
 		}
+		apt_log(MPF_LOG_MARK, APT_PRIO_INFO, "RTP Stream settings: Codecs own preference: %d",
+			rtp_stream->settings->own_preferrence);
+
+		const char *codec_list1_str = NULL;
+		char codec_list1_str_buf[2048];
+		const char *codec_list2_str = NULL;
+		char codec_list2_str_buf[2048];
+
+		// Log both codec lists
+		if (mpf_codec_list_to_string(codec_list1_str_buf, sizeof(codec_list1_str_buf), codec_list1) > 0)
```

Comment:
Why not fold it inside the intersect, we know this test fails. Also, should we do a dump after capabilities prune?